tools/cpp/windows_cc_configure.bzl

```diff
@@ -232,16 +232,12 @@ def find_vc_path(repository_ctx):
     # 5. Check default directories for VC installation
     auto_configure_warning_maybe(repository_ctx, "Looking for default Visual C++ installation directory")
     for path in [
-        "Microsoft Visual Studio\\2019\\Preview\\VC",
-        "Microsoft Visual Studio\\2019\\BuildTools\\VC",
-        "Microsoft Visual Studio\\2019\\Community\\VC",
-        "Microsoft Visual Studio\\2019\\Professional\\VC",
-        "Microsoft Visual Studio\\2019\\Enterprise\\VC",
-        "Microsoft Visual Studio\\2017\\BuildTools\\VC",
-        "Microsoft Visual Studio\\2017\\Community\\VC",
-        "Microsoft Visual Studio\\2017\\Professional\\VC",
-        "Microsoft Visual Studio\\2017\\Enterprise\\VC",
         "Microsoft Visual Studio 14.0\\VC",
+        "Microsoft Visual Studio\\2019\\Preview\\VC",
```

Comment:
Currently we need 2022\Preview\VC to build google/or-tools ^^;
or you could add it to below comprehension list.
Also order may matter i.e. prefer "Preview" before Enterprise before Community etc...

Reply:
in previous code the order was "Preview", "BuildTools", "Community", "Professional", "Enterprise". Should I keep that?

Reply:
Also, I guess adding `"Microsoft Visual Studio\\2017\\Preview\\VC"` is harmless even though that did not exist AFAIK, right?

Reply:
(for the moment I preserved the order prior to this patch wrt editions, and added the 2017\Preview one, but that can still be tweaked)